### PR TITLE
Add hex logo to repo

### PR DESCRIPTION
**Pull request by kaitejohnson**

### CODE REVIEW

**Comment by seabbs** [comment]
love to see it. make sure to you `usethis::use_logo()` and check with @dylanhmorris about CDC logo rules (I am just guessing there are some).

**Comment by kaitejohnson** [comment]
Good idea, didn't realize that existed! Will check with Kayla and Isaac

**Comment by dylanhmorris** [comment]
I'm afraid I don't know anything offhand about logo rules / policies (if any), but will investigate

**Comment by athowes** [comment]
Can you add code to generate the hex plot? Incase you need to change it in future.

**Comment by kaitejohnson** [comment]
> Can you add code to generate the hex plot? Incase you need to change it in future.

@athowes I didn't make it, Kayla English generated it in illustrator! I think it's ok for it not to be auto-updated.

**Comment by damonbayer** [comment]
@kaitejohnson if it's generated in illustrator, can she provide an svg for us?

**Comment by kaitejohnson** [comment]
Not loving the positioning now but can play modify the geometry or the title text.

**Comment by dylanhmorris** [comment]
> Not loving the positioning now but can play modify the geometry or the title text.

I would check the dimensions of the `.svg`. Looks like there might be some blank space above and below the hex.

**Comment by gvegayon** [comment]
I just saw this, @kaitejohnson. I LOVE IT! I think that once the CDC rules for this sort of stuff are clear, we are good to go!

**Comment by kaitejohnson** [comment]
@gvegayon Thank Kayla English!!

**Comment by dylanhmorris** [comment]
@kaitejohnson I think this needs the changes from `main` pulled in so that the required checks will run.

File: man/figures/wwinference-logo.png
Comment (dylanhmorris) [inline man/figures/wwinference-logo.png:1]: delete this?

File: man/figures/wwinference-logo.svg
Comment (dylanhmorris) [inline man/figures/wwinference-logo.svg:1]: delete this also?

**Review by dylanhmorris — CHANGES_REQUESTED** [review]
Need to delete old logos

**Review by dylanhmorris — APPROVED** [review]
LGTM!

**Pull request merged**